context.cc

```diff
@@ -0,0 +1,122 @@
+#include "context.h"
+#include "context-macros.h"
+#include "deps/include/v8-template.h"
```

Comment:
Please move the V8 headers to a separate section. (In more places.)

Reply:
Like this?

```
#include <sstream>

#include "deps/include/v8-exception.h"
#include "deps/include/v8-message.h"
#include "deps/include/v8-primitive.h"

#include "errors.h"
#include "utils.h"
```

I think it should be consistent now. Of course there are some cases like this, where some includes are only used for C++ code, some necessary for Go code.

```
#include "errors.h"

#ifdef __cplusplus

#include "deps/include/v8-persistent-handle.h"
```

Reply:
Thanks, yepp. Yeah, I noticed the ifdefs. Ugly, but I get the reason. We could separate the ifdefs for the includes and for the declarations, but I think it's readable enough.

Re. ordering, the Google C++ styleguide has one section for standard library headers, one for non-local headers, and one for the local project's headers. The one exception is that the module's own header file (i.e. `value.h` in 
`value.cc`) comes before all other headers, because doing that will catch issues with the header not including its own dependencies properly.